web/styles.css

```
@@ -517,7 +517,266 @@ h1 {
+    line-height: 1.5;
+    color: var(--text-color);
+    min-height: 200px;
+    max-height: 400px;
+}
+
+.log-entry {
+    padding: 4px 8px;
+    display: flex;
+    align-items: flex-start;
+    gap: 8px;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)